VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/api/VTCareResource.java

```diff
@@ -20,4 +23,30 @@ public Provider createProvider(Provider provider) throws Exception {
             throw new Exception("An exception occurred while persisting the provider details.");
         }
     }
+
+    @POST
+    @Path("login")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response login(String json) throws Exception {
+        JsonObject o = new Gson().fromJson(json, JsonObject.class);
```

Comment:
Again, we should name our variables in a meaningful way. Please rename o to a something that is appropriate in the context.